docroot/themes/humsci/humsci_basic/src/scss/components/_views-exposed-form.scss

```diff
@@ -7,25 +7,40 @@
   .form-item {
     width: 100%;
     margin-bottom: hb-calculate-rems(24px);
+
     @include grid-media-min('md') {
       width: calc(50% - #{hb-calculate-rems(18px)});
       margin-right: hb-calculate-rems(18px);
     }
-    @include grid-media-min('lg') {
-      width: calc(33% - #{hb-calculate-rems(18px)});
+
+    .hb-layout-builder-main-content & {
```

Comment:
hmm is this the right class to target? maybe this is better? `hb-three-column__main`

Reply:
That makes sense to me! The `hb-layout-builder-main-content` class was used before so that's why I re-used it. But this makes more sense as `hb-three-column__main` and `hb-three-column__sidebar-1` sit beside each other in the DOM.